manager/notifier.go

```diff
@@ -325,6 +326,59 @@ func (n *notifier) sendSlackNotification(op notificationOp, config *pb.SlackConf
 	return nil
 }
 
+// https://www.flowdock.com/api/team-inbox
+// compulsory fields in Flowdock JSON: source, from_address, subject, content
+// Content-type: application/json
+// POST to https://api.flowdock.com/v1/messages/team_inbox/:flow_api_token
+type FlowdockMessage struct {
```

Comment:
AFAICS this type doesn't need to be exported / capitalized (but the fields need to be, for JSON unmarshaling).